src/test/java/bio/terra/app/configuration/WebConfigTest.java

```diff
@@ -0,0 +1,51 @@
+package bio.terra.app.configuration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import bio.terra.common.category.Unit;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
+
+@Tag(Unit.TAG)
+class WebConfigTest {
+
+  @Test
+  void swaggerUiVersionMatchesClasspath() throws IOException {
+    PathMatchingResourcePatternResolver resolver = new PathMatchingResourcePatternResolver();
+    Resource[] resources =
+        resolver.getResources("classpath:/META-INF/resources/webjars/swagger-ui-dist/*/");
+
+    Pattern versionPattern = Pattern.compile(".+/swagger-ui-dist/(\\d+\\.\\d+\\.\\d+)/");
+    Optional<String> currentVersion =
+        Arrays.stream(resources)
+            .map(
+                resource -> {
+                  Matcher matcher =
+                      versionPattern.matcher(((ClassPathResource) resource).getPath());
+                  return matcher.matches() ? matcher.group(1) : null;
+                })
+            .filter(v -> v != null)
```

Comment:
```suggestion
            .filter(Objects::notNull)
```

Another approach is to `flatMap()` and return `Stream.of()` for the `null` case.